charts/charts/openebs-crds/Chart.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v2
 name: openebs-crds
-version: 4.1.0
+version: 4.1.1
```

Comment:
Is this version bump necessary?

Reply:
Not really, but iirc we decided that we would follow the chart version for this.